.gitignore

```diff
@@ -34,3 +34,6 @@ tsconfig.tsbuildinfo
 
 # Unsupported lock files
 package-lock.json
+
+# npm config file
+.npmrc
```

Comment:
Very slim chances, but if someone is using his hardcoded npm token on local npmrc file, there is potential for doing commit in git repo by accident, so usually good practice to ignore this file

Reply:
This can apply to every dot file in the project. As every other dot file `.npmrc` contains project level configuration related to project which every contributor must follow. And for such configuration I don't think ignoring is a good practice.

Reply:
I don't believe we ignore npmrc in 1.x.